src/analyses/uninit.ml

```diff
@@ -28,7 +28,8 @@ struct
   let should_join x y = D.equal x y
 
   let startstate v : D.t = D.empty ()
-  let otherstate v : D.t = D.empty ()
+  let threadenter ctx f args : D.t = D.empty ()
+  let threadspawn ctx f args fctx = D.bot ()
```

Comment:
`D.bot ()` and `D.empty ()` are the same for `ValueDomain.AddrSetDomain`, so maybe use `D.bot ()` here instead of `D. empty ()`

Reply:
I used `D.empty ()` for `threadenter` because the previous `otherstate` used that. And I used `D.bot ()` for `threadspawn` because that more clearly expresses the notion that when `threadspawn` is joined with its parallel transfer function, it will not affect that.

Reply:
Alright!